back-end/Models/followSystem/Following.js

```diff
@@ -0,0 +1,17 @@
+const jwt_decode = require("jwt-decode");
+const connection = require("../../db");
+const followUser = (req, res) => {
+  const token = req.headers.authorization.split(" ").pop();
+  const decoded = jwt_decode(token);
```

Comment:
instead of using these two lines, in the auth middleware, you could set `req.token` to equal the parsed token to use it directly without having to write these lines

Reply:
Done